golang/connection.go

```diff
@@ -175,6 +178,10 @@ func newConnection(conn net.Conn, initialState connectionState, handlers *handle
 		framePool = DefaultFramePool
 	}
 
+	connID := atomic.AddUint32(&nextConnID, 1)
+	log.SetPrefix(fmt.Sprintf("C%v ", connID))
```

Comment:
Thanks, I totally missed that. I've created a prefixLogger which can be used to wrap loggers with a specific prefix instead.